tests/asts/test_set_facts.yml

```diff
@@ -32,10 +32,10 @@
               name: playbooks/validate_args_playbook.yml
               extra_vars:
                 my_vars:
-                  fact.alpha: 1
-                  fact.beta.location: "{{ Naboo }}"
-                  fact.beta.name: R2D2
-                  fact.msg: hello world
+                  event.alpha: 1
```

Comment:
@bzwei Does it make sense to not call this event, maybe something else like group. Because we already have lot of confusion between event/fact events/facts and this example adds to that confusion. These are all user defined variables so we can call it anything we want.

Reply:
@mkanoor We have to construct keys under my_vars exactly the same as what are under ansible_eda in order for this [playbook](https://github.com/ansible/ansible-rulebook/blob/main/tests/playbooks/validate_args_playbook.yml#L2) to validate the results by set_facts.